sql/bench_test.go

```diff
@@ -54,7 +52,7 @@ func benchmarkCockroach(b *testing.B, f func(b *testing.B, db *sql.DB)) {
 }
 
 func benchmarkPostgres(b *testing.B, f func(b *testing.B, db *sql.DB)) {
```

Comment:
Include comments about the necessary postgresql.conf settings in the code, not just the commit message.